### PR TITLE
Upgrade matomo

### PR DESCRIPTION
**Pull request by hellais**
In this PR I am upgrading to the latest matomo.

This fixes:
* https://github.com/ooni/sysadmin/issues/225
* https://github.com/ooni/sysadmin/issues/149

Moreover I use this as a chance to also improve the letsencrypt role and make a new `docker` role that works well on modern versions of debian.

It also includes some fixes to issues encountered in dom0-bootstrap as part of the deploy.

### CODE REVIEW

File: ext/gh-droplets.json [hunk 1]
```diff
@@ -1,1693 +1,1693 @@
 {
```
Comment (bassosimone) [inline ext/gh-droplets.json]: what is this file?

File: ansible/roles/docker/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    cache_valid_time: 28800
+
+- name: adds Docker repository key
+  apt_key: >
+    id=9DC858229FC7DD38854AE2D88D81803C0EBFCD88
+    url="https://download.docker.com/linux/debian/gpg"
+
+- name: adds Docker repository and update apt cache
+  apt_repository: >
+    repo="deb https://download.docker.com/linux/debian {{ ansible_distribution_release }} stable"
```
[the remark this answers is not in the document]
Comment (hellais) [reply ansible/roles/docker/tasks/main.yml]: It's an ansible variable which is set the to the distribution name (ex. `buster`). Search docs.ansible.com for `ansible_distribution_release` to find more information about it.

File: ansible/group_vars/all/vars.yml [hunk 3]
```diff
@@ -33,12 +33,12 @@ passwd:
   plpsql:   {login: plpsql, comment: Pipeline Postgresql}
   plshovel: {login: plshovel, comment: Pipeline Shovel}
 
-  nodeexp:  {login: nodeexp, comment: Prometheus Node Exporter}
+  nodeexp:  {login: nodeexp, comment: Prometheus Node Exporter} # DONE
```
[the remark this answers is not in the document]
Comment (hellais) [reply ansible/group_vars/all/vars.yml]: These are some notes for me so I remember which usernames have been migrated.

File: ext/gh-droplets.json (hunk 1 above)
Comment (hellais) [reply ext/gh-droplets.json]: It's generated by `ext-inventory.yml` and it has a dump of all the machines running in our greenhost account.

**Pull request merged**